streams/src/main/java/org/apache/kafka/streams/processor/DefaultPartitionGrouper.java

```diff
@@ -61,7 +61,7 @@ protected int maxNumPartitions(Cluster metadata, Set<String> topics) {
             List<PartitionInfo> infos = metadata.partitionsForTopic(topic);
 
             if (infos == null)
-                throw new KafkaException("topic not found :" + topic);
+                throw new KafkaException("Topic not found during partition assignment: " + topic);
```

Comment:
Interesting. In that case it doesn't seem to important in the short term, it's only relevant once the exceptions become user facing.